src/main/java/com/assignment1/item/ItemEntity.java

```diff
@@ -0,0 +1,94 @@
+package com.assignment1.item;
+
+import com.assignment1.Type;
+
+/** Class as blueprint for Item Entity. */
+@SuppressWarnings("PMD.DataClass")
+public class ItemEntity {
+
+  /** Name of item. */
+  private String name;
+  /** type(RAW, MANUFACTURED, IMPORTED) of item. */
+  private Type type;
+  /** quantity of item. */
+  private int quantity;
+  /** price of item. */
+  private double price;
+  /** tax of item. */
+  private double tax;
+
+  /** Constructor initializing default values. */
+  public ItemEntity() {
+    name = "";
+    type = Type.RAW;
+    quantity = 1;
```

Comment:
default constructors shouldn't set these values

Reply:
```suggestion
    type = Type.UNKNOWN;
    quantity = 0;
```

Reply:
Incorporated this change.